File: Python/algorithmic_toolbox/01_a_plus_b.py
import sys


def add_two_numbers(a, b):
    return a + b


def main():
    lines = sys.stdin
    a = int(next(lines))
    b = int(next(lines))
    result = add_two_numbers(a, b)
    print(result)


if __name__ == "__main__":
    main()

File: Rust/algorithmic_toolbox/src/bin/01_a_plus_b.rs
use std::io;

fn add_two_numbers(a: i32, b: i32) -> i32 {
    a + b
}

fn read_int() -> i32 {
    let mut x = String::new();
    io::stdin().read_line(&mut x).unwrap();
    x.trim().parse().unwrap()
}

fn main() {
    let x = read_int();
    let y = read_int();
    let result = add_two_numbers(x, y);
    println!("{}", result);
}
